INSTALL.rst

```diff
@@ -49,12 +56,30 @@ Create a virtual environment and clone the INSPIRE source code using `git`:
 
 .. code-block:: console
 
-    $ mkvirtualenv inspirehep
+    $ mkvirtualenv -p /usr/bin/python2.7 inspirehep
```

Comment:
It looks like you can also do `--python=python2.7`, which looks a bit more robust.